### DISCUSSION TITLE
[1 febbraio 2024] Esercizio 5

### DISCUSSION THREAD

**Discussion by FeddyLix17** [sapienzastudentsnetwork/psd2223#267] [Esami]
Usando teoremi ed assiomi dell’algebra di Boole, semplificare la seguente espressione

<div align="center">

![image](https://github.com/user-attachments/assets/cdd22776-971c-42bd-83bd-e3c60e158d08)

</div>

**Comment by Nw4pS** [comment]
![SmartSelect_20240907_191243_Samsung Notes.jpg](https://github.com/user-attachments/assets/5ad69707-cef2-4c32-be78-83e680fedd69)
